server/src/data.js

```diff
@@ -359,24 +359,37 @@ module.exports = class Data {
 
 	async checkDuplicates(table, url, title = null) {
 		const res = await this.db.query(`SELECT * FROM ${table}`);
+		
+		// Helper function to clean string
+		const cleanString = (str) => str.replace(/[\s-_]/g, '');
+	
 		let similar = res.rows.find(col => {
-			let urlDist = new Levenshtein(col.url, url);
-			if(urlDist.distance <= 1) {
+			// Clean the URL and title strings before calculating Levenshtein distance
+			let cleanedColUrl = cleanString(col.url);
+			let cleanedUrl = cleanString(url);
+			
+			let urlDist = new Levenshtein(cleanedColUrl, cleanedUrl);
+			if(urlDist.distance === 0) { // Changed the threshold to 0
```

Comment:
Isn't Levenshtein dictance 0 equivalent to using == ?
If we keep this distance at 0, we may just want to check `cleanedColUrl.toLowerCase() === cleanedUrl.toLowerCase()` and remove the levensthein dependency completely.

Reply:
You're right! Just fixed.